test/message/encryptMessage.spec.ts

```diff
@@ -74,11 +79,16 @@ describe('encryptMessage', () => {
 
     it('it does not compress a message by default', async () => {
         const decryptedPrivateKey = await decryptPrivateKey(testPrivateKeyLegacy, '123');
-        const { message: encrypted, sessionKey } = await encryptMessage({
+        const sessionKey = {
+            data: hexToUint8Array('c5629d840fd64ef55aea474f87dcdeef76bbc798a340ef67045315eb7924a36f'),
+            algorithm: enums.read(enums.symmetric, enums.symmetric.aes256)
+        };
```

Comment:
This is fairly nitpicky but could we have a call to `generateSessionKey` here instead, to show how it "should" be done? (Idem in the other ones)

Reply:
I can add an end-to-end test, but I prefer the encryptMessage tests to not depend on other functions for the most part 👍 
I'll add usage examples in the readme/other docs anyway.

Reply:
Alrighty :+1: